src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -768,6 +767,10 @@ public void testMaxWarnings() throws IOException {
                 .isTrue();
     }
 
+    private static List<String> readAllFile(File outputFile) throws IOException {
+        return Files.readAllLines(outputFile.toPath(), StandardCharsets.UTF_8);
```

Comment:
@rnveach , should it be "readWholeFile" ? to be English correct.

Reply:
@pbludov , I do not see how it is related  to modernization ?

looks like regular refactoring to reuse util method.

Reply:
If I understood the idea correctly, the modernizer finds API calls that can be replaced with similar ones from the JDK.

For this particular case, we can avoid usage of [commons-io](https://github.com/gaul/modernizer-maven-plugin/blob/master/modernizer-maven-plugin/src/main/resources/modernizer.xml#L795).

This is a good idea in general. But there are some issues. For example, modernizer blindly suggests to replace `InnutableSet.of` from `guava` with `Set.of` from JDK.
Even for `EnumSet`, which is handled efficiently by guava, but not by pure JDK.

Reply:
I would say correct english is `readWholeFile`, `readFile`, or as the command inside it is so named, `readAllLines`.

Reply:
> For example, modernizer blindly suggests to replace `InnutableSet.of` from `guava` with `Set.of` from JDK

It depends on java level set for plugin. I set it to 8, so it should not suggest things like `Set.of(...)`

Reply:
@strkkk, please rename method.
And move refactoring to separate commit.

Reply:
It is not clear why this plugin forced you to do this refactoring

Reply:
Output from modernizer:
````
[INFO] --- modernizer-maven-plugin:2.3.0:modernizer (modernizer) @ checkstyle ---
[ERROR] \checkstyle\src\main\java\com\puppycrawl\tools\checkstyle\api\Violation$Utf8Control.java:531: Prefer java.io.InputStreamReader(java.io.InputStream, java.nio.charset.Charset)
[ERROR] \checkstyle\src\main\java\com\puppycrawl\tools\checkstyle\checks\header\AbstractHeaderCheck.java:116: Prefer java.io.InputStreamReader(java.io.InputStream, java.nio.charset.Charset)
````

No violations came from `CheckstyleAntTaskTest`.